registry/registry-mesh/src/main/java/com/alipay/sofa/rpc/registry/mesh/MeshRegistry.java

```diff
@@ -228,8 +229,8 @@ protected void doUnRegister(String serviceName, ProviderInfo providerInfo) {
 
         UnPublishServiceRequest unPublishServiceRequest = new UnPublishServiceRequest();
         unPublishServiceRequest.setServiceName(serviceName);
+        unPublishServiceRequest.setProperties(providerInfo.getStaticAttrs());
```

Comment:
Why we use `getStaticAttrs` in unPublishServiceRequest ,but use `getParameters` in subscribeRequest. Should we put both staticAttrs and  params into pub/unpub properties ?